Gemfile

```diff
@@ -36,6 +36,8 @@ end
 
 if RUBY_VERSION >= '1.9' && RUBY_VERSION <= '2.1'
   gem 'rubocop', "~> 0.23.0"
+  # The latest version (2.2.1) breaks on ruby 1.9.2
+  gem 'rainbow', "~> 2.1.0" if RUBY_VERSION == "1.9.2"
```

Comment:
Instead, why don't we limit rubocop to running only on 2.0?  I don't think we even need to run rubocop on multiple ruby versions.